### PR TITLE
Drop common package

### LINKED ISSUE #2693: Drop common package
And move all modules to robottelo main package. Import from robottelo.helpers does not hurt the meaning and is shorter than robottelo.common.helpers.

### PR DESCRIPTION
**Pull request by elyezer**
Modules from the common package is not placed under the robottelo root
package. This makes package names shorter.

Closes #2693

### CODE REVIEW

**Comment by Ichimonji10** [comment]
Nice.

    $ git grep ' common$'
    robottelo/performance/graph.py:    remaining parts like labels, titles, and rendering are both common
    robottelo/ui/locators.py:    # common
    $ git grep robottelo.common

**Comment by Ichimonji10** [comment]
I was able to sucessfully execute a few tests with each of `make test-foreman-api`, `make test-foreman-cli` and `make test-foreman-ui`.

ACK.

**Comment by elyezer** [comment]
@Ichimonji10 I forgot to mention that I have run all robottelo tests and also make docs both work properly. Have run some tests to make sure it does not break things and got the same results as yours.

**Comment by Ichimonji10** [comment]
:+1:

**Comment by elyezer** [comment]
Rebased and resolved conflicts.

**Comment by elyezer** [comment]
Another rebase

**Comment by elyezer** [comment]
Updated based on the changes introduced by #2730

File: robottelo/decorators.py [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 import random
-import requests
 import unittest
-
-from ddt import data as ddt_data
 from functools import wraps
-from robottelo.common import conf
-from robottelo.common.constants import NOT_IMPLEMENTED
 from xml.parsers.expat import ExpatError, errors
 from xmlrpclib import Fault
```
Comment (sthirugn) [inline robottelo/decorators.py]: Why are there two sets of import/from statements? They can be merged right?

**Comment by sthirugn** [comment]
ACK

**Pull request merged**